model/src/main/proto/oppia_logger.proto

```diff
@@ -118,7 +118,45 @@ message EventLog {
   }
 }
 
-// Structure of a container that contains event logs.
-message EventLogs{
+// Structure for an exception log.
+message ExceptionLog {
+  // The timestamp of the exception.
+  int64 timestamp = 1;
+  // The stack trace of the exception.
+  int64 stacktrace = 2;
+  // The message of the exception.
+  string message = 3;
+  // The case of the exception.
+  string cause = 4;
+  // The type of exception.
+  ExceptionType exception_type = 5;
+
+  enum ExceptionType {
+    // The undefined exception type.
+    CRASH_TYPE_UNSPECIFIED = 0;
+    // The exception which is causes the application to close.
+    FATAL = 1;
+    // The exception which does not cause the application to close.
+    NON_FATAL = 2;
+  }
+}
+
+// Structure for a container that contains an event log report.
+message OppiaEventLogs {
   repeated EventLog event_log = 1;
 }
```

Comment:
I am not entirely sure what is happening here. WE are making a list of OppiaEventLogs which has only a repeated list of event log..and OppiaLog is not being used anywhere? Please recheck the entire storage solution. There are a lot of semantic issues here

Reply:
Modified, PTAL.